cmd/zetaclientd/encrypt_tss.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"crypto/sha256"
+	"encoding/json"
+	"errors"
+	"github.com/spf13/cobra"
+	"io"
+	"os"
+)
+
+var encTssCmd = &cobra.Command{
+	Use:   "tss-encrypt",
+	Short: "Utility command to encrypt existing tss key-share file",
+	RunE:  EncryptTSSFile,
+}
+
+type TSSArgs struct {
+	secretKey string
+	filePath  string
+}
+
+var tssArgs = TSSArgs{}
+
+func init() {
+	RootCmd.AddCommand(encTssCmd)
+
+	encTssCmd.Flags().StringVar(&tssArgs.secretKey, "secret", "", "tss-encrpyt --secret p@$$w0rd")
+	encTssCmd.Flags().StringVar(&tssArgs.filePath, "filepath", "", "tss-encrpyt --filepath ./file.json")
```

Comment:
Why not using positional argument instead?

Reply:
I think it is more clear what the arguments are like this but it does work either way. I can change it to positional if necessary.

Reply:
As a general rule, if it is required, positional is more adapted